### ISSUE TITLE
Log Persistence Walkthrough Files Empty

### ISSUE THREAD

**Issue by dasxx17** [tektoncd/dashboard#2185]
# Describe the bug

Following https://github.com/tektoncd/dashboard/blob/main/docs/walkthrough/walkthrough-logs.md, despite the files being written to the bucket, when downloading the files they are all empty.  I have tried modifying the format of the file as well to no avail, and I also enabled --stream-logs=true (not sure if this is necessary) in my dashboard deployment.  All of the FluentD logging and operator pods all seem to be healthy (I followed the troubleshooting guide).  Interestingly, the files all seem to be of appropriate size (e.g 560 bytes,  1.2 kilobytes, etc.).  However, they are empty when I download them.  Not sure if I am missing any configuration, but I followed the walkthrough step by step.

TLDR:  Log files all being written to S3 bucket, files are all empty despite appropriate number of bytes in file.  I see the logs are appearing in my Tekton Dashboard as well.  

# Expected behaviour

Logs should be filled with the sample (Log Line 1, 2, etc.)

# Steps to reproduce the bug

Follow https://github.com/tektoncd/dashboard/blob/main/docs/walkthrough/walkthrough-logs.md

# Environment details

- Kubernetes version: 1.18
- Cloud-provider/provisioner: EKS
- Versions:
   - Tekton Dashboard: 0.18.1
   - Tekton Pipelines: 0.26.0
   - Tekton Triggers: 0.15.0
- Install namespaces:
   - Tekton Dashboard: tekton-pipelines
   - Tekton Pipelines: tekton-pipelines
   - Tekton Triggers: tekton-pipelines

**Comment by dasxx17** [comment]
CC @AlanGreene

**Comment by AlanGreene** [comment]
Thanks for reporting @dasxx17. I think this was an oversight when the external logs feature was added, it looks like the open / download links have never worked for external logs.

I'll work on a fix, should be a relatively small change 🤞

**Comment by dasxx17** [comment]
Hi @AlanGreene not sure if this is related to the lack of the open/download links feature for external logs, but when configuring the fallback I am not seeing the logs appear again in my dashboard.  I have the external-logs flag configured as per the tutorial:
` - --external-logs=http://logs-server.tools.svc.cluster.local:3000/logs`

But I am still seeing the `Unable to fetch logs` message when browsing tasks.  I have verified that the logs are indeed appearing in my S3 bucket Minio server.  All the pods are looking healthy (operator, server, fluentbit, fluentd), with the exception of some restarts on the server pod `Normal  Pulled     61s                  kubelet            Container image "node:14" already present on machine`.

I do not believe this would impact it, but I did deploy an ALB Ingress Controller to expose the `logs-server` deployment.  Thanks for your help!
